psamm/datasource/sbml.py

```diff
@@ -692,6 +692,23 @@ def _create_unique_id(self, in_dict, orig_id):
                 suffix += 1
         return orig_id
 
+    def _add_flux_bounds(self, r_id, model, flux_limits, equation):
```

Comment:
Doesn't `add` flux bounds but returns something... maybe call it `_get_flux_bounds` or change it to actually set the `lowerFluxBound`, `upperFluxBound` properties.